Neos.Neos/Classes/Fusion/AbstractMenuItemsImplementation.php

```diff
@@ -84,7 +84,7 @@ public function getItems()
     {
         if ($this->items === null) {
             $fusionContext = $this->runtime->getCurrentContext();
-            $this->currentNode = isset($fusionContext['activeNode']) ? $fusionContext['activeNode'] : $fusionContext['documentNode'];
+            $this->currentNode = $fusionContext['activeNode'] ?? $fusionContext['documentNode'];
```

Comment:
Can `activeNode` be dropped? It is never used elsewhere…

Reply:
Also stumbled upon this lately and haven't found a usage of it.